### PR TITLE
Add Swift language support

### PR DESCRIPTION
**Pull request by Terky**

### CODE REVIEW

**Comment by rohitpaulk** [comment]
Thanks @Terky! Tests look good, will review & deploy on Monday 🙂

**Comment by Terky** [comment]
> Thanks @Terky! Tests look good, will review & deploy on Monday 🙂

Sorry that it took me so long to get back and fix it. Glad to be back! 
P.S. Other challenges starters are coming :)

**Comment by rohitpaulk** [comment]
Woot, can't wait!

File: compiled_starters/sqlite-starter-swift/Sources/swift-sqlite-challenge/Main.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+@main
+struct Main {
+
```
Comment (rohitpaulk) [inline compiled_starters/sqlite-starter-swift/Sources/swift-sqlite-challenge/Main.swift:5]: @Terky quick question - is it intentional to leave an extra space below the `struct Main {` line here? Same for line 40 (`extension UnsignedInteger {`)? 

Is this is a widely accepted code style for Swift? Just curious, haven't seen this elsewhere.

**Comment by rohitpaulk** [comment]
@Terky you can now try this here: https://app.codecrafters.io/courses/sqlite?fresh=true. 

Also, I've bumped you to a VIP user so that you have free access to all content :) Thanks so much for this!

**Pull request merged**